fe/src/main/java/org/apache/doris/load/loadv2/SparkLoadJob.java

```diff
@@ -909,4 +949,30 @@ private void initTDescriptorTable(DescriptorTable descTable) {
             tDescriptorTable = descTable.toThrift();
         }
     }
+
+    public Set<Long> getTableWithBitmapColumn() {
+        if (tableWithBitmapColumn.size() != 0) {
```

Comment:
I think we can prepare this bitmap table map when analyzing the job, and then persist it.

Reply:
👌